packages/fx-core/resource/package.nls.json

```diff
@@ -809,6 +809,7 @@
   "driver.apiKey.log.skipUpdateApiKey": "Skip updating API key as the same property exists.",
   "driver.apiKey.log.successUpdateApiKey": "API key updated successfully!",
   "driver.apiKey.confirm.update": "The following parameters will be updated:\n%s\nDo you want to continue?",
+  "driver.apiKey.info.update": "API key updated successfully! The following parameters have be updated:\n%s",
```

Comment:
Please change 'have be updated' to 'have been updated'

Reply:
Updated, thanks!